### PR TITLE
cost: autoscaling for suggestion executor and gateway

### PR DESCRIPTION
**Pull request by IyadhKhalfallah**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

**Review by azayz — APPROVED** [review]

File: now/constants.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from now.utils import BetterEnum
 
 NOW_GATEWAY_VERSION = '0.0.4-fix-custom-gateway-remote-4'
-NOW_PREPROCESSOR_VERSION = '0.0.124-refactor-spot-instances-1'
+NOW_PREPROCESSOR_VERSION = '0.0.124-feat-autoscale-suggestion-0'
```
Comment (Tanguyabel) [inline now/constants.py:8]: I don't think you need to rebuild the image to apply the cloud settings?

Comment (IyadhKhalfallah) [reply now/constants.py:8]: @Tanguyabel Actually it failed without this (You can check the previous commit). I think I also need to update the other versions as well

**Comment by IyadhKhalfallah** [comment]
> Can you check that the state of the autocomplete executor is persisted?

@joschkabraun we do now, nice catch 🚀  still some tests are failing for some reason.

**Review by joschkabraun — APPROVED** [review]
Nice, LGTM!

**Pull request merged**